baybe/objectives/single.py

```diff
@@ -70,6 +72,21 @@ def to_botorch(self) -> MCAcquisitionObjective:
 
         return IdentityMCObjective()
 
+    @override
+    def to_botorch_posterior_transform(self) -> ScalarizedPosteriorTransform:
+        if not (
+            isinstance((t := self._target), NumericalTarget)
+            and ((tr := t.transformation).is_affine)
+        ):
+            raise NonGaussianityError(
+                f"Converting an objective of type '{type(self).__name__}' is only "
+                f"possible when the transformation result is Gaussian, that is, "
+                f"when the target is of type '{NumericalTarget.__name__}' and the "
+                f"assigned transformation is affine."
+            )
+
+        return (tr if not t.minimize else tr.negate()).to_botorch_posterior_transform()
```

Comment:
would prefer to read that

```suggestion
        return (tr.negate() if t.minimize else tr).to_botorch_posterior_transform()
```